libs/common/src/platform/server-notifications/internal/default-server-notifications.service.ts

```diff
@@ -62,21 +63,46 @@ export class DefaultServerNotificationsService implements ServerNotificationsSer
     private readonly authRequestAnsweringService: AuthRequestAnsweringServiceAbstraction,
     private readonly configService: ConfigService,
   ) {
-    this.notifications$ = this.accountService.activeAccount$.pipe(
-      map((account) => account?.id),
-      distinctUntilChanged(),
-      switchMap((activeAccountId) => {
-        if (activeAccountId == null) {
-          // We don't emit server-notifications for inactive accounts currently
-          return EMPTY;
-        }
+    this.notifications$ = this.configService
+      .getFeatureFlag$(FeatureFlag.InactiveUserServerNotification)
+      .pipe(
+        switchMap((inactiveUserServerNotificationEnabled) => {
+          if (inactiveUserServerNotificationEnabled) {
+            return this.accountService.accounts$.pipe(
+              map((accounts) => Object.keys(accounts) as UserId[]),
+              switchMap((userIds) => {
```

Comment:
Another one in between here. If it's effectively the same user ids I'd love to not to the reconnect. i.e `["user1", "user2"] === ["user2", "user1"]` in this case.

```suggestion
              map((accounts) => Object.keys(accounts) as UserId[]),
              distinctUntilChanged(comparatorFunction),
              switchMap((userIds) => {
```

Reply:
fixed with [865ffa2](https://github.com/bitwarden/clients/pull/16151/commits/865ffa2739417b3458748eb6e35a74fc764a912c)